test/infrastructure/docker/config/crd/bases/infrastructure.cluster.x-k8s.io_dockermachinepools.yaml

```diff
@@ -195,7 +195,12 @@ spec:
     storage: false
     subresources:
       status: {}
-  - name: v1alpha4
+  - additionalPrinterColumns:
```

Comment:
AFAIK if there are no `additionalPrinterColumns` the CRD gets an age column automatically, is the idea here that we add the age column everywhere in case we add others printer columns later?

Reply:
There is no way to do it automatically (it's only added per default if no `additionalPrinterColumns` are configured).
Some additional context: https://github.com/kubernetes-sigs/cluster-api/pull/5180#discussion_r699066958

So yes, the idea is to add it everywhere now, so we don't forget to add it later.